### ISSUE TITLE
Travis error on swig python binding.

### ISSUE THREAD

**Issue by miurahr** [miurahr/gdal#11]
```
running build_ext
Traceback (most recent call last):
  File "setup.py", line 339, in <module>
    **extra )
  File "/usr/lib/python2.7/distutils/core.py", line 151, in setup
    dist.run_commands()
  File "/usr/lib/python2.7/distutils/dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "/usr/lib/python2.7/distutils/dist.py", line 972, in run_command
    cmd_obj.run()
  File "/usr/lib/python2.7/distutils/command/build.py", line 128, in run
    self.run_command(cmd_name)
  File "/usr/lib/python2.7/distutils/cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "/usr/lib/python2.7/distutils/dist.py", line 971, in run_command
    cmd_obj.ensure_finalized()
  File "/usr/lib/python2.7/distutils/cmd.py", line 109, in ensure_finalized
    self.finalize_options()
  File "setup.py", line 214, in finalize_options
    self.gdaldir = self.get_gdal_config('prefix')
  File "setup.py", line 188, in get_gdal_config
    return fetch_config(option)
  File "setup.py", line 141, in fetch_config
    raise gdal_config_error, e""")
  File "<string>", line 4, in <module>
__main__.gdal_config_error: [Errno 2] No such file or directory
make[2]: *** [gdal/swig/python/CMakeFiles/gdal_python] Error 1
make[1]: *** [gdal/swig/python/CMakeFiles/gdal_python.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 94%] Built target frmts_raw
make: *** [all] Error 2

```

**Comment by miurahr** [comment]
fixed by 488dd5718db576f2ab807519180cd761e1bbf080

**Comment by miurahr** [comment]
fixed again in ebe7ebc
